src/components/Chart/Bars.js

```diff
@@ -333,7 +333,7 @@ const BarChart = props => {
           inlineValueUnit && inlineValueUnit,
           inlineLabel && d.datum[inlineLabel]
         ].join(' ')
-        d.inlineLabelTextWidth = labelGauger(d.inlineLabelText)
+        d.inlineLabelTextWidth = labelGauger(d.inlineLabelText || '')
```

Comment:
Darn didn't notice that. But IMO we shouldn't even call labelGauger when we have nothing:
```suggestion
        d.inlineLabelTextWidth = d.inlineLabelText
          ? labelGauger(d.inlineLabelText)
          : 0
```

Reply:
sure, I just used an existing pattern in the code XD